src/main/interface/settings.c

```diff
@@ -761,6 +761,7 @@ const clivalue_t valueTable[] = {
     { "gps_rescue_throttle_hover",  VAR_UINT16 | MASTER_VALUE, .config.minmax = { 1000, 2000 }, PG_GPS_RESCUE, offsetof(gpsRescueConfig_t, throttleHover) },
     { "gps_rescue_sanity_checks",   VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_GPS_RESCUE }, PG_GPS_RESCUE, offsetof(gpsRescueConfig_t, sanityChecks) },
     { "gps_rescue_min_sats",        VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 50 }, PG_GPS_RESCUE, offsetof(gpsRescueConfig_t, minSats) },
+    { "gps_rescue_min_dth",         VAR_UINT16  | MASTER_VALUE, .config.minmax = { 50, 32000 }, PG_GPS_RESCUE, offsetof(gpsRescueConfig_t, minRescueDTH) },
```

Comment:
I'm not too sure about the units in C, but I think that an uint8_t can hold until 32000. The type of var must be changed.

Reply:
@McGiverGim no, uint8_t is 0-255.

Reply:
Agree, should be coherent in both places. I think it's ok to make it max 255, don't see why you would want a MIN dth beyond that.

Reply:
Fixed.